src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -144,7 +144,6 @@ OverlayImpl::OverlayImpl (
     BasicConfig const& config)
     : Overlay (parent)
     , io_service_ (io_service)
-    , work_ (boost::in_place(std::ref(io_service_)))
```

Comment:
It was nice to set up `work_` in the ctor-initializer, is there a way to do it? I won't hold up the PR for it though.